dawn/src/dawn/Optimizer/PassStageReordering.cpp

```diff
@@ -23,17 +22,13 @@
 
 namespace dawn {
 
-PassStageReordering::PassStageReordering(OptimizerContext& context, ReorderStrategy::Kind strategy)
-    : Pass(context, "PassStageReordering"), strategy_(strategy) {
-  dependencies_.push_back("PassSetStageGraph");
-}
-
 bool PassStageReordering::run(
-    const std::shared_ptr<iir::StencilInstantiation>& stencilInstantiation) {
+    const std::shared_ptr<iir::StencilInstantiation>& stencilInstantiation,
+    const Options& options) {
   const std::string filenameWE =
       fs::path(stencilInstantiation->getMetaData().getFileName()).filename().stem();
-  if(context_.getOptions().ReportPassStageReodering)
-    stencilInstantiation->jsonDump(filenameWE + "_before.json");
+
+  // stencilInstantiation->jsonDump(filenameWE + "_before.json");
```

Comment:
Are these dumps not useful anymore? (also the "_after.json" one)

Reply:
I added a `write-si` option that writes these to file.